azure-spring-boot/src/main/resources/META-INF/spring.factories

```diff
@@ -1,4 +1,3 @@
-org.springframework.boot.env.EnvironmentPostProcessor=com.microsoft.azure.spring.cloudfoundry.environment.VcapProcessor,\
```

Comment:
do not remove `org.springframework.boot.env.EnvironmentPostProcessor`. It should looks like

```
org.springframework.boot.env.EnvironmentPostProcessor=\
com.microsoft.azure.keyvault.spring.KeyVaultEnvironmentPostProcessor,\
com.microsoft.azure.spring.autoconfigure.sqlserver.AlwaysEncryptedEnvironmentPostProcessor
```

Reply:
Added it